### ISSUE TITLE
KeyError due to type inconsistency at parse_order of qryptos/quoine in Python

### ISSUE THREAD

**Issue by xeno14** [ccxt/ccxt#1090]
Could you help on this Python specific issue?

### Issue
`parse_order` of qryptos/quoine always raise `KeyError` therefore `create_order` currently not works.

```python
import ccxt

key={
  "apiKey": "",
  "secret": ""
}
q = ccxt.quoine(key)
# create_order fails at parse_order
res = q.create_order(symbol="QASH/JPY",type="limit",side="buy",price=1,amount=1)
```

```
Traceback (most recent call last):
  File "test.py", line 9, in <module>
    print(q.create_order(symbol="QASH/JPY",type="limit",side="buy",price=1,amount=1))
  File "/usr/local/lib/python3.5/dist-packages/ccxt/qryptos.py", line 219, in create_order
    return self.parse_order(response)
  File "/usr/local/lib/python3.5/dist-packages/ccxt/qryptos.py", line 234, in parse_order
    market = self.marketsById[marketId]
KeyError: 50
```
### Cause
`fetch_markets` returns `id` as string, `marketsById` works for string keys. On the other hand, `privatePostOrders` returns `product_id` is integer -- `marketsById` does not accept it.
 
```python
q = ccxt.quoine()
q.load_markets()
q.marketsById["50"]  # it works
q.marketsById[50]   # it does not work
```
I have tested Javascript as well -- js does not care string or integer so this issue is specific to Python (I'm not sure about PHP).

```js
const ccxt = require ('ccxt');
(async () => {
    let q = new ccxt.quoine();
    await q.load_markets () // request markets
    console.log(q.marketsById[50] == q.marketsById["50"]); // true
}) ()
```

### Solution for Python
Just make sure `product_id` is string using `str()` like [this commit](https://github.com/xeno14/ccxt/commit/1f1b5f87ba68d1e10d84620004517d0d4029b81d) on my forked repository. However, this change cannot be a solution because Python code is automatically generated from Javascript. I tried to change Javascript code as below, but auto-generated Python code was not valid Python (`toString` remains in Python code...)
Could you consider solving this issue?

```javascript
// in parseOrder
let marketId = order['product_id'].toString();
```

### Environment

- OS: ArchLinux
- Programming Language: Python 3.5/3.6
- CCXT version: 1.10.575
- Exchange: qryptos,quoine
- Method: parseOrder

**Comment by kroitor** [comment]
Hi!

Thanks for this comprehensive report!

> I tried to change Javascript code as below, but auto-generated Python code was not valid Python (toString remains in Python code...)

You almost did it right! You just missed one symbol, violating the rules from here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes. This particular rule: *every opening bracket like ( or { should have a space before it!*.

So the line should be

```
let marketId = order['product_id'].toString (); // ←- see the space before the bracket? )
```

Unfortunately, the transpiler is very sensitive and we use the spacing to hint it. We don't have a better tool for now and it does the job at least, if we follow each rule literally.

> Could you consider solving this issue?

Well, if you want, you can now submit a proper PR and I would merge it right away. What do you think about it? Or do you want me to do it?

**Comment by xeno14** [comment]
Thank you very much for quick reply and correcting my code. I will submit a pull-request soon later!

**Comment by kroitor** [comment]
Fixed in #1091
